From 0cbf823912b74140e3db276d08e5807d4500b350 Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Mon, 20 Nov 2023 10:00:10 +0200
Subject: [PATCH] [SW-156330] gaudi2: increase arm ppboot size

As part of tpm support for gaudi3, tpm is moved to
common so gaudi2 and gaudi3 can reuse maximum code.
gaudi3 arm ppboot is still getting compiled and will
be removed in separate jira, until then increase
arm ppboot size for compilation to go through.
In separate jira all gaudi2 arm code should be removed
and this change will become redundant anyway.

Change-Id: I5aee6607d9891da3f5758dacd4b81c68f760ce30
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/arch/gaudi2/arm/Makefile b/arch/gaudi2/arm/Makefile
index f7346f0..9064b11 100644
--- a/arch/gaudi2/arm/Makefile
+++ b/arch/gaudi2/arm/Makefile
@@ -1,7 +1,7 @@
 include $(HL_DEV_ARCH_COMMON)/Makefile
 
 PHY_FW_SIZE := 0x5400
-PPBOOT_MAX_SIZE := 20480
+PPBOOT_MAX_SIZE := 32768
 
 CFLAGS += -mstrict-align
 image_validation: image_validation_common
diff --git a/arch/gaudi2/arm/common.h b/arch/gaudi2/arm/common.h
index 33f957e..4d3e02d 100644
--- a/arch/gaudi2/arm/common.h
+++ b/arch/gaudi2/arm/common.h
@@ -32,7 +32,7 @@
 #define HL_FW_BOOT_PTRS_BASE		0x60000
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
 
-#define PPBOOT_SIZE			0x6000		/* 24KB */
+#define PPBOOT_SIZE			0x8000		/* 32KB */
 #define PREBOOT_SIZE			0x20000		/* 128KB */
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
 
